src/domain/signupGroup/eventInfo/EventTimes.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Accordion from '../../../common/components/accordion/Accordion';
+import LoadingSpinner from '../../../common/components/loadingSpinner/LoadingSpinner';
+import { EventFieldsFragment } from '../../../generated/graphql';
+import useLocale from '../../../hooks/useLocale';
+import skipFalsyType from '../../../utils/skipFalsyType';
+import { getEventFields } from '../../event/utils';
+import { getEventTimeStr } from '../../events/utils';
+import useEventWithSubEventsData from '../hooks/useEventWithSubEventsData';
+import styles from './eventInfo.module.scss';
+
+export type EventTimesProps = {
+  event: EventFieldsFragment;
+};
+
+const EventTimes: FC<EventTimesProps> = ({ event }) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+
+  const { eventWithSubEvents, loading: isLoadingEvent } =
+    useEventWithSubEventsData({
+      id: event.id,
+      superEventType: event.superEventType,
+    });
+
+  if (!event.superEventType) {
+    return null;
+  }
+
+  return (
+    <Accordion
+      className={styles.eventTimesAccordion}
+      toggleButtonLabel={t('signup.eventTimes')}
+    >
+      <LoadingSpinner isLoading={isLoadingEvent}>
+        <ul>
+          {eventWithSubEvents?.subEvents.filter(skipFalsyType).map((item) => {
```

Comment:
Same comment here as in the linkedregistrations-ui PR: should we ensure ordering by `startTime` through sorting? Since ordering by ID might not actually be in chronological order.

Reply:
Added sorting